packages/amino/src/signer.ts

```diff
@@ -1,6 +1,6 @@
-import { StdSignature } from "@cosmjs/amino";
-
-import { StdSignDoc } from "./encoding";
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why is this newly added here?

Reply:
Thanks, `StdSignDoc` was here for a part of the transition.